lib/stat_tracker.rb

```diff
@@ -142,6 +142,51 @@ def worst_team(season)
     end.first
   end
 
+  def total_game_teams(filtered_game_teams = @game_teams)
+    filtered_game_teams.count
+  end
+
+  def avg_score(filtered_game_teams = @game_teams)
+    ratio(total_score(filtered_game_teams), total_game_teams(filtered_game_teams))
+  end
+
+  def team_id_to_team_name(id)
+    team_obj = @teams.select do |team|
+      team.team_id == id
+    end
+    team_obj[0].team_name
+  end
```

Comment:
I did already make this method, so it's up to you if you wanna keep it. (for reference: it's "team_names_by_team_id")

Reply:
hahah same here, let's keep all them and then refactor later